storage/database/leveldb_bench_common_test.go

```diff
@@ -147,13 +147,13 @@ func Benchmark_read_data(b *testing.B) {
 
 // benchmarkFIFOCacheGetParallel measures the performance of the fifoCache when reading data in parallel
 func benchmarkFIFOCacheGetParallel(b *testing.B, valueSize int) {
-	cache := common.NewCache(common.FIFOCacheConfig{CacheSize: numDataInsertions})
+	cache := common.NewCache(common.FIFOCacheConfig{CacheSize: numDataInsertions, IsScaled: true})
```

Comment:
@winnie-byun Please remove `IsScaled` from benchmark also.

Reply:
@KimKyungup There are `cache.scale` and `cache.level` instead of `IsScaled` in benchmark. I think they are useful for scaled cache.
I will add comment instead of removing them.

Reply:
Applied your suggestion after reconsidering.